src/components/ReportPage/index.jsx

```diff
@@ -228,9 +249,23 @@ class ReportPage extends PureComponent {
     ));
   }
 
+  renderSearches() {
```

Comment:
We only have one Search in the UI and I also don't think we plan to have more than one. How about we rename this method to `renderSearch`?